### ISSUE TITLE
Option to disable translation creation for a kata

### ISSUE THREAD

**Issue by Voileexperiments** [Codewars/codewars.com#1955]
There is a checkbox for "allow contributors" for katas in kata editor. I think there should be one for "allow translations" too. Here's why:

The translation system right now is too inclined on letting translations be created and approved. What if I *don't* want translations to a kata because it only makes sense in that language? What if I want to prevent other people shitpost translations (like, making CoffeeScript/TypeScript translations of advanced JS feature katas)? If I don't reject the translations, someone will come and approve it one day. If I reject it, now I might have to answer a million questions why I rejected the translations, and that still doesn't stop someone from forking/writing one from scratch again. It's a continuous nuisance for some of the katas around.

**Comment by docgunthrop** [comment]
Perhaps one possible approach could be to filter the "Choose Language" list at the translation page so it only includes languages that are approved by the author (or, alternatively, languages that have not been blacklisted by the author).

In order to do that, however, you'd then likely have to make some UI changes to the Kata editor page (and then some) so the author can check off their approved languages.

**Comment by Steffan153** [comment]
@docgunthrop Yeah, that would be helpful for some kata, but for most, the "Allow translations" should be enough. Plus, that would be a lot less work. :smile:

**Comment by nomennescio** [comment]
I think preventing people from making translations because the author MIGHT in the future add them himself is a VERY bad reason to prevent translation being added. In that way authors can 'hoard' translation points, or prevent others from gaining points. Especially for new languages, it is useful to be able to at least start translating some simple existing Katas.
